Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
```

Comment:
Yeah, I'm not really sure. All the flask documentation that I've seen uses `from flask import ...`, which is why we've stuck with it. I kind of like the explicitness of the first approach, though I agree that it becomes a little cumbersome with a lot of functions. Does PEP have a preference?